SmartDeviceLink/SDLPermissionManager.h

```diff
@@ -84,6 +89,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)removeObserverForIdentifier:(SDLPermissionObserverIdentifier)identifier;
 
+
+/**
+ *  Check whether or not an RPC needs encryption.
+ */
+- (BOOL)rpcRequiresEncryption:(__kindof SDLRPCMessage *)rpc;
```

Comment:
Nicole's comment isn't quite right. It should be `SDLPermissionRPCName` like other APIs in the `SDLPermissionManager`. It should not be `SDLRPCMessage` because that requires an overly verbose and complicated API. `SDLPermissionRPCName` isn't a perfect solution, but it's consistent with the other permission manager APIs and can be documented well and aligns relatively well with Android's usage of `FunctionID` which contains both the `FunctionID` and the function name.